easybuild/main.py

```diff
@@ -579,30 +580,23 @@ def process_eb_args(eb_args, eb_go, cfg_settings, modtool, testing, init_session
     return overall_success
 
 
-def main(args=None, logfile=None, do_build=None, testing=False, modtool=None):
+def main(args=None, logfile=None, do_build=None, testing=False, modtool=None, eb_go=None, cfg_settings=None,
+         init_session_state=None, hooks=None):
```

Comment:
no need to pass down `hooks`, and it's probably better/cleaner to only introduce a single new argument to `main`, like `prepared_cfg_data` which includes `init_session_state`, `eb_go`, and `cfg_settings`

fixed in https://github.com/XavierCS-dev/easybuild-framework/pull/1

Reply:
Merged